### DISCUSSION TITLE
Personal Access Token (PAT) - unable to copy/paste into window terminal

### DISCUSSION THREAD

**Discussion by taramaj** [community/community#103155] [New to GitHub]
Hi,
I have created a PAT and used it on the windows command line as needed i.e. with push command.  The problem is that I have to manually input every character of the PAT for it to be accepted.  If I try to paste the PAT into the command line, I get a failure message:

remote: Support for password authentication was removed....
remote: Please see....
fatal: Authentication failed....

I've pasted and copied the PAT to/from a word doc and Notebook++.  I looked for whitespaces as well and tried to eliminate them.  

Any help would be appreciated!  Thank you!

**Comment by gil-son** [comment]
Hi

Try to do that:
```
git push -u origin main "your_personal_access_token"
```

**Comment by taramaj** [comment]
Thank you for the response.  Unfortunately, this did not work.

**Reply by gil-son** [reply]
clone your project 
```
git clone https://{token}@github.com//project.git
```

Into directory of the project:
```
git config --global credential.helper cache
```



OR

```
git remote set-url origin https://{your_username}:{your_new_token}@github.com/{repo_path}.git
```
